web/src/components/CleanLink.tsx

```diff
@@ -1,9 +1,38 @@
+import React, { ReactNode } from 'react'
 import styled from 'styled-components'
 import { Link } from 'react-router-dom'
+import { isExternalUrl } from '../utils/UrlCheck'
 
-const CleanLink = styled(Link)`
+const StyledCleanLink = styled(Link)`
   color: inherit;
   text-decoration: none;
 `
+const StyledCleanAnchor = styled.a`
+  color: inherit;
+  text-decoration: none;
+`
+
+interface CleanLinkProps {
+  to: string
+  children: ReactNode
```

Comment:
Maybe it makes sense to allow an array here as well, then we don't need to wrap everything in a fragment if using this component.

Reply:
not necessary